etc/environment.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
   - pytest
   - pytest-xdist
   - pytest-cov
+  - mfpymake
```

Comment:
Hi,  Thanks for your help so far.  You are right to tread a bit carefully, this is my first pull request to a widely used repository.  Sorry in advance for my learning process :). 

I included mfpymake as when I ran the tests on the environment I made from etc/environment.yml (via conda) there were many import errors from pymake.  I found it a bit tricky to debug as I assumed that the etc/... files held all dependencies for testing.  Happy to remove, but it would be helpful to include as a requirement for in autotest/Autotesting.md

Reply:
It would be good to remove all mentions of mfpymake.  We provide executables in many different forms here: https://github.com/MODFLOW-USGS/executables.